src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({
+  post: { username, title, body, user, comment },
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        { `Posted by ${user.username}` }
+        {' '}
+        <UserInfo user={user} key={user.id} />
```

Comment:
Looks weird, UserInfo component already shows user name, so no need to duplicate it inside PostInfo component